kafka/hbase-kafka-proxy/src/main/java/org/apache/hadoop/hbase/kafka/KafkaBridgeConnection.java

```diff
@@ -161,6 +145,10 @@ public RegionLocator getRegionLocator(TableName tableName) throws IOException {
     return null;
   }
 
+  @Override
+  public void clearRegionLocationCache() {
```

Comment:
what happens when building against HBase 2.1.z here?

Reply:
Compilation error because of the `@Override` annotation.

Reply:
Can we some how make things work with both? Can we just leave off the Override annotation?